nixos/modules/services/torrent/flood.nix

```diff
@@ -56,7 +63,7 @@ in
             cfg.host
             "--port"
             (toString cfg.port)
-            "--rundir=/var/lib/flood"
+            "--rundir=${cfg.stateDir}"
```

Comment:
Since this service is `DynamicUser`, you'll likely want to bindmount this state directory as readwrite, since it will get a different UID/GID on each run.